vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -727,22 +727,91 @@ forms: a_form_name"
 )
 ```
 
+# Merging forms in the `secuTrialdata` object
+
+Naturally, you will sometimes need to merge/join some of the data from the individual form data stored
+in your `secuTrialdata` object. To achieve this you can use the `merge()` function.
+A common task may be to merge repetition form data to its parent form.
+In our case `esurgeries` can be naively merged with `baseline` via the `mnpdocid`:
+
+\vspace{5pt}
+```{r}
+bl_surg <- merge(x = ctu05_data$baseline, y = ctu05_data$esurgeries, by = "mnpdocid")
```

Comment:
does it make sense to do as.data.frame before this?

Reply:
maybe show a simple 1:1 merge before going into the more complex merges?

Reply:
We've been using this way to access the data throughout the vignette (also my preferred way) so I'd like to keep this one "as is".

Reply:
ok regarding as.data.frame.
Any thoughts about the 1:1 merge?

Reply:
Ah, didn't see that. I'll have a look if I can find something where we merge data from e.g. the same visit.

Reply:
I would also note that the primary key is the `mnppid`, which is unique to an individual (rather than a visit or form). If you have data collected once from different visits (`mnpcvpid` might then be different), then you would need `mnppid`. 

The randomization example you give, the rando result is equally relevant to the other visits in the baseline form too (many participants have multiple visits), so to me, the `mnppid` would be the natural choice to merge be here (for an n:1 merge). For a simple 1:1 merge, you could use the `studyterminat` form...

Reply:
it's maybe also good to mention mnpaid (which contains the IDs as they are shown in secutrial)

Reply:
At this point I would prefer not to elaborate more on the topic. We state that a full explanation of all `mnp` vars and the full linkage explanation is beyond the scope of the vignette so I think it should be fine.

Reply:
ok, you get the general idea across, so it's ok